public/js/main.js

```diff
@@ -1,22 +1,124 @@
 'use strict';
-
-// Alert user about comment submission success
-// Get the parameter from the URL
-// Disable linter for this line because web environment
-// isn't disabling node checks
-const urlParams = new URLSearchParams(window.location.search);	// eslint-disable-line node/no-unsupported-features/node-builtins
-const msg = urlParams.get('message');
+// function for toggling modal
 const modal = document.querySelector('.modal');
 const toggleModal = () => {
 	modal.classList.toggle('show-modal');
-};
+}
+// event listener for clicking out of modal
 const windowOnClick = (event) => {
 	if (event.target === modal) {
 		toggleModal();
 	}
-};
-if (msg) {
-	window.addEventListener('click', windowOnClick);
-	document.getElementById('modal-text').innerText = msg;
-	toggleModal();
-}
\ No newline at end of file
+}
+window.addEventListener('click', windowOnClick);
+
+// Handle form submit
+const commentForm = document.getElementById('commentForm');
+
+commentForm.addEventListener('submit', function (e) {
+	e.preventDefault();
+	const formData = new FormData(this);
+	const searchParams = new URLSearchParams();
+
+	for (const pair of formData) {
+		searchParams.append(pair[0], pair[1]);
+	}
+	
+	fetch('/addComment', {
+		method: 'POST',
+		body: searchParams
+	}).then(function (response) {
+		// if successfull, add comment to comment section
+		return response.json();
+	}).then(function (data) {
+		if (data.msg) {
+			document.getElementById('modal-text').innerText = data.msg;
+			toggleModal();
+		}
+		const commentsSection = document.getElementById('comments');
+		commentsSection.innerHTML = data.commentsHTML;
+	}).catch(function (error) {
+		console.error(error);
```

Comment:
No modal with a "sorry" message?

Reply:
The modal pops up with an error message if an error message is sent from the server.